### ISSUE TITLE
Inline SVG is marked as an invalid tag

### ISSUE THREAD

**Issue by johnduffell** [structured-data/linter#23]
Hi, when I try to validate this document, I get some errors:
```html
<!DOCTYPE html>
<html><head><title>hi</title></head>
	<body>
		<svg width="400" height="200">
			<circle cx="150" cy="100" r="50" />
		</svg>
	</body>
</html>
```
```
Linter Messages
validation http://example.org/: Tag svg invalid
validation http://example.org/: Tag circle invalid
```
As far as I can tell it is valid.  Am I missing something?

**Comment by gkellogg** [comment]
The linter uses [Nokogiri](https://github.com/sparklemotion/nokogiri) to do HTML parsing and validation, and there seems to be an open issue on this particular subject: https://github.com/sparklemotion/nokogiri/issues/1243. I'll investigate further to see if there's a way to silence such issues, as the main purpose is to look for illegal use of HTML elements and attributes, and bad wellformedness.

In the mean time, if it can't be easily resolved otherwise, a comment that polyglot markup may be marked in Linter Messages may be all we can do.

**Comment by johnduffell** [comment]
Oh oops I did find that issue by googling and assumed it was unrelated!  Seems like noone is working on that one, so I'll ignore it in our tests if nothing else.  I'm currently working on calling linter directly locally and I may create a PR for you if I have something useful to contribute back.

**Comment by gkellogg** [comment]
HTML validation is done in the rdf-rdfa gem, and it may require some configuration there (https://github.com/ruby-rdf/rdf-rdfa/blob/develop/lib/rdf/rdfa/reader/nokogiri.rb#L283).

I think it may be possible to configure when parsing, which is done here: https://github.com/ruby-rdf/rdf-rdfa/blob/develop/lib/rdf/rdfa/reader/nokogiri.rb#L180.

**Comment by gkellogg** [comment]
Fixed in rdf/rdfa/#3.
